### ISSUE TITLE
API provider still showing up on Connections page 

### ISSUE THREAD

**Issue by dongniwang** [syndesisio/syndesis#4003]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## The problem

The API provider still shows up on the Home, Create a Connection wizard, and Connections page as a regular connection. 

Thought this is fixed via https://github.com/syndesisio/syndesis/issues/3889, but I'm using 1.5.6 and it's still showing up. 

## Expected behavior

It should not show up on those pages. 

## Screenshot

<img width="1406" alt="screen shot 2018-10-30 at 3 58 50 pm" src="https://user-images.githubusercontent.com/24943812/47746589-bbc2ae00-dc5c-11e8-81cb-74147457c40c.png">

<img width="924" alt="screen shot 2018-10-30 at 3 59 16 pm" src="https://user-images.githubusercontent.com/24943812/47746609-c8df9d00-dc5c-11e8-8568-79afa030d89a.png">

<img width="1421" alt="screen shot 2018-10-30 at 3 59 40 pm" src="https://user-images.githubusercontent.com/24943812/47746648-d85ee600-dc5c-11e8-9d04-a9f49aa2c61b.png">

cc: @gashcrumb @nicolaferraro @riccardo-forina

**Comment by gashcrumb** [comment]
Yeah, [this commit](https://github.com/syndesisio/syndesis/commit/baf8cdce309daa44521bf658d6da02f08e960565) and the one after it should have done the trick, the 1.5.x branch was created some time after the PR was merged.  Also note that log, timer and webhook should also not be showing up either.  I'm not sure why these would be showing up now.

**Comment by dongniwang** [comment]
I think the Log, Timer and Webhook connections are OOTB ones? So it's fine for them to show up?

**Comment by gashcrumb** [comment]
No, they should only show up in the editor as they're system provided
connections.

**Comment by gashcrumb** [comment]
Can you hit me up with a link to your install today?  I wouldn't mind taking a look.

**Comment by dongniwang** [comment]
This issue was fixed once I switched to an incognito window. It seems like it's going on for quite a while now that after application updates, people are seeing the oder version of it until they manually refresh the page or clear the cache. 

Tempory remedy: 
- Clear browser cache 
- OR try with an incognito window

Users might encounter this as well. Consider providing an app level banner with action so users can perform the refesh. 

cc: @gashcrumb @syndesisio/uxd
